documentation/feature/medical-system.md

```diff
@@ -6,238 +6,370 @@ group: feature
 
-Should a unit be in pain, materializing itself with a chromatic aberration screen effect, he can be given morphine.
+#### 2.1.1 Wounds, bandages and medications
 
-### 2.2 Basic medical system - recommended gear
-* Soldier:
-  * 10 x Bandage (basic)
+<p align="center">
+<b>Wounds</b>
+</p>
 
-* Medic:
-  * 15-25 x Bandage (basic)
-  * 6 x Blood IV (500ml)
-  * 10 x Morphine Autoinjector
-  * 10 x Epinephrine Autoinjector
+It's pretty straightforward compared to advanced, you only have two types of wounds.
 
-## 3. Advanced Medical
-The advanced medical system provides a more complex and detailed medical simulation and is based off the CSE/CMS medical system. It focuses on a more realistic model for injuries and treatment, thus resulting in a more important and prominent role for combat medics, and a bigger incentive to not get shot.
+- Yellow: you need one bandage to heal it.
+- Red: you need two bandages to heal it.
 
-The system behind advanced medical is designed to attempt to mimic important parts of the human body, as well as react to any injuries sustained and treatments applied in a realistic manner. The available treatments and supplies in advanced medical are based off the Tactical Combat Casualty Care (TCCC) guidelines, which are the same guidelines used by real-life combat medics around the world.
+<p align="center">
+<b>Bandages</b>
+</p>
 
-Besides the 4 elements introduced by basic medical, advanced introduces the following:
+- All of them have the same effect.
 
-* More detailed wound system
-* Accurate blood loss based upon sustained injuries
-* Vitals, including heart rate and blood pressure
-* Cardiac Arrest
-* Various treatment methods such as CPR, different kinds of IVs and tourniquets
-* A basic medication simulation
+<p align="center">
+<b>Tourniquet</b>
+</p>
 
-### 3.1 How it works
+- Serves no use in basic
 
-Same as with basic, when hit an injury is sustained. Different though is that the type of injury and the severity of it are based upon how the damage was done and what caused it. This affects both blood loss and immediate consequences, such as being knocked out or being killed right away. When a player has sustained an injury, this will be indicated by flashing red of the screen; this means the player is bleeding.
+<p align="center">
+<b>IVs</b>
+</p>
 
-#### 3.1.1 Stopping bleeding
-In order to stop the bleeding, all injuries on every bodypart requires treatment. This is done by either applying a tourniquet to legs or arms as a temporarly solution, or by using bandages to stop the bleeding as a more permament fix. 
+IV | Effect
+---------- | ---------- |
+Saline | Serves no use in basic
+Plasma | Serves no use in basic
+Blood | Restore the blood of the patient
```

Comment:
`Restores`
Concerning blood in basic, maybe add why there are different amounts of blood 250ml, 500ml, 1000ml